user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Accessing_custom_apps.md

```diff
@@ -6,6 +6,10 @@ uid: Accessing_custom_apps
 
 The DataMiner Low-Code Apps are available on the DataMiner landing page. To access this page, browse to the IP or hostname of a DMA or to `https://[Your DMA]/root`, depending on your configuration.
 
+You can then access any of the available low-code apps by clicking the relevant app icon under *Other apps*, or [create a new low-code app](xref:Creating_custom_apps).
```

Comment:
Not necessarily under _Other apps_ since we can have other sections on the home page.